src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -190,30 +190,29 @@ const ChartsHolder = (props: {
                       target="_blank"
                       rel="noopener noreferrer"
                     >
-                      Studies
+                      Experts recommend
                     </a>{' '}
-                    suggest that in order to successfully contain COVID, 80% of
-                    contacts for each new COVID case must be traced within 48
-                    hours in order to contain the spread of infection. Experts
-                    estimate that tracing each new case within 48 hours requires
-                    an average of 10 contact tracers per new case, as well as
-                    fast testing.{' '}
+                    that 80% of contacts for each new case must be traced within
+                    48 hours in order to contain COVID. Experts estimate that
+                    tracing each new case within 48 hours requires an average of
+                    10 contact tracers per new case, as well as fast testing.
+                    Our contact tracing data is sourced from{' '}
                     <a
-                      href="https://blog.covidactnow.org/modeling-metrics-critical-to-reopen-safely/"
+                      href="https://testandtrace.com/"
```

Comment:
these got moved to the metric/contact_tracing.ts file

Reply:
i think this can't move into `metric/contact_tracing.ts` because of hyperlinks